analytics.go

```diff
@@ -143,7 +142,7 @@ func New(key string) *Client {
 		now:      time.Now,
 		uid:      uid,
 	}
-
+	go c.loop()
```

Comment:
You're right, I was thinking of merging this into the `v3.0` branch first and not master so it shouldn't break anything (same for all the stuff we discussed today).